dataset_eval/gaia_wrapper.py

```diff
@@ -44,9 +46,29 @@ def train_model(project_dir, groundtruth_file, filelist_file):
         datasets_dir=os.path.join(project_dir, "datasets"),
         results_dir=os.path.join(project_dir, "results"),
     )
+    """
+    Converting the preferences from string type to list
+    """
+    c_value = [int(value) for value in c_value.split(',')]
+    gamma_value = [int(s) for s in gamma_value.split(',')]
+    preprocessing_values = [str(value) for value in preprocessing_values.split()]
+    """
+    Updating C, Gamma, and preprocessing values in the project file which were
+    taken as input by the user
+    """
+    update_parameters(project_file, c_value, gamma_value, preprocessing_values)
     run_tests(project_file)
     return select_best_model(project_dir)
 
+def update_parameters(project_file, c_value, gamma_value, preprocessing_values):
+    with open(project_file, "r") as pfile:
+        project = yaml.load(pfile)
+    for pref in project['classifiers']['svm']:
+        pref['gamma'], pref['C'], pref['preprocessing'] = gamma_value, c_value, \
```

Comment:
I don't think there's a reason that this has to be a multi-assignment statement. Three separate statements is a bit easier to follow